models/cow_protocol/tx_hash_labels/bluechip_investment/cow_protocol_tx_hash_labels_bluechip_investment.sql

```diff
@@ -0,0 +1,8 @@
+{{
+    config(
+        alias='tx_hash_labels_bluechip_investment',
+        post_hook='{{ expose_spells_hide_trino(\'["ethereum"]\', "sector", "tx_hash_labels", \'["gentrexha"]\') }}'
+    )
+}}
+
+SELECT * FROM {{ ref('cow_protocol_tx_hash_labels_bluechip_investment_ethereum') }}
```

Comment:
nit:
```suggestion
SELECT * FROM {{ ref('cow_protocol_tx_hash_labels_bluechip_investment_ethereum') }}

```

Reply:
I can't seem to spot the difference between the two. Can you elaborate on that?

Reply:
New line at EOF!